addon_fabfile.py

```diff
@@ -606,13 +607,9 @@ def install_addon(name,
                     with open(filepath, 'w') as f:
                         f.write(filedata)
 
-                    # Do a syncdb, to add the add-on's models. (This can't be
+                    # Do a migration to add the add-on's models. (This can't be
                     # done before INSTALLED_APPS is updated.)
-                    error = "doing a syncdb."
-                    _django_manage("makemigrations --noinput " + name,
-                                   proj_settings=settings,
-                                   install_dir=install_dir)
-
+                    error = "doing a migrate."
                     _django_manage("migrate --noinput",
                                    proj_settings=settings,
```

Comment:
also need to do a `manage.py loaddata` call to address any fixtures that might be present.